cli/cmd/org/edit.go

```diff
@@ -73,7 +75,7 @@ func EditCmd(cfg *config.Config) *cobra.Command {
 		},
 	}
 	editCmd.Flags().SortFlags = false
-	editCmd.Flags().StringVar(&orgName, "org", cfg.Org, "Organization name")
+	editCmd.Flags().StringVar(&orgName, "name", cfg.Org, "Organization name")
```

Comment:
Actually i got the couple comments on this to have a same flag name for every subcommand, it was previously `--name` for `org create` and `--org` for others. an example of some comments is in bug bash https://www.notion.so/rilldata/Rill-cloud-Bug-bash-2413071d9b734a9285ba677369874b69?pvs=4#8e9f0c91579040a298b9bbe0e3a47c73

Reply:
Also i guess with `rill org` having flag as `--name` looks better than `rill org delete --org`. We have `--org` with all the commands for project i.e `rill project --org`

So i guess we can keep it `--name` for org cmd

Reply:
I think with the new positional support, calling the flag `org` everywhere except `create` feels most consistent. Since that's what it's called everywhere. Similarly, `rill project edit` would take `--project` to override the name, not `--name`. 

It's fine to have `rill org create` as an exception since here the name is not yet acting as an ID.